### DISCUSSION TITLE
deepin_V23_beta3】【非常严重】【treeland】技术预览版界面，触摸板点击打开软件没有反应，没有鼠标没测试鼠标，可以使用两个按键打开

### DISCUSSION THREAD

**Discussion by jiutian321** [linuxdeepin/developer-center#7133] [Troubleshooting | 故障排查]
### SUMMARY | 问题概要

deepin_V23_beta3】【非常严重】【treeland】技术预览版界面，触摸板点击打开软件没有反应，没有鼠标没测试鼠标，可以使用两个按键打开，播放音乐失败，安装mpv可以播放视频音乐

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

```plain
deepin_V23_beta3 ddm/treeland预览技术版

treeland | 0.2.2



### PRE-REQUIREMENT | 预置条件

_No response_

### STEPS TO REPRODUCE | 复现步骤

1.使用treeland界面必现，

### OBSERVED RESULT | 观察到的结果

技术预览版界面，触摸板点击打开软件没有反应，没有鼠标没测试鼠标，可以使用两个按键打开，播放音乐失败，

### EXPECTED RESULT | 期望的结果

能正常操作

### ADDITIONAL INFORMATION | 额外补充

_No response_
```[tasklist]
### Tasks
```

**Comment by BLumia** [comment]
> 技术预览版界面，触摸板点击打开软件没有反应，没有鼠标没测试鼠标，可以使用两个按键打开，播放音乐失败，安装mpv可以播放视频音乐

触摸板点击哪里打开软件，dde-launchpad？两个按键打开是什么意思，哪两个按键，键盘按键？具体哪个键？播放音乐失败是哪个软件播放失败？深度音乐？浏览器？别的播放器？

如果您不是测试人员的话，非常建议您在【深度之家】应用内进行问题反馈而不是在 GitHub 进行反馈，测试人员会引导您补充对应的信息。缺失太多信息是完全无助于问题的处理的 :joy:

**Comment by jiutian321** [comment]
@BLumia 触摸板点击所有地方，如桌面快捷方式，开始菜单中的快捷方式，任务栏上的都点不动，两个按键就是笔记本触摸板下面不是有两个按键么，相当于鼠标左键，鼠标右键，深度音乐软件打不开，我论坛传了个日志，可以看看，这里好像传不了

好的

**Comment by jiutian321** [comment]
预览技术版下 深度之家提示网络错误，打不开，其实是有网的 @BLumia

**Comment by BLumia** [comment]
> 触摸板点击所有地方，如桌面快捷方式，开始菜单中的快捷方式，任务栏上的都点不动，两个按键就是笔记本触摸板下面不是有两个按键么，相当于鼠标左键，鼠标右

这个是触控板驱动/配置问题，和 treeland 无关。

> 深度音乐软件打不开，我论坛传了个日志，可以看看，这里好像传不了

这里可以传的，复制粘贴文本也可以。以及请确保每一个不同的问题汇报成单独的 issue 而不是一股脑报到同一个 issue 里。

> 预览技术版下 深度之家提示网络错误，打不开，其实是有网的

又是一个完全不同的问题。ping 的通baidu吗？

------

建议切回 x11 在深度之家反馈吧。这个 issue 如果你还希望补充信息的话，我就转到 discussion 继续讨论吧。
